src/app/modules/item/services/item-datasource.service.ts

```diff
@@ -52,7 +52,7 @@ export class ItemDataSource implements OnDestroy {
 
     ).subscribe(state => this.state.next(state));
 
-    this.subscription = this.userSessionService.session$.subscribe(_s => this.refreshItem());
+    this.subscription = this.userSessionService.userChanged$.subscribe(_s => this.refreshItem());
```

Comment:
I assume we could get rid of this subscription by using `repeatLatestWhen(this.userSessionService.userChanged$)` on the the previous operation... but that out of scope of this PR.